lib/ddtrace/contrib/gc/integration.rb

```diff
@@ -0,0 +1,34 @@
+require 'ddtrace/contrib/integration'
+require 'ddtrace/contrib/gc/configuration/settings'
+require 'ddtrace/contrib/gc/patcher'
+
+# Try to load the native extension, but it's okay if it fails.
+require 'ddtrace/ddtrace' rescue LoadError
+
+module Datadog
+  module Contrib
+    module GC
+      class Integration
+        include Contrib::Integration
+
+        register_as :gc
+
+        def self.version
+          Gem::Version.new('1.0.0')
```

Comment:
Not really sure what we should be looking at to determine a version. Maybe the Ruby version? 🤷‍♂️

Reply:
`version` is just an abstract way of asking "what is the version of the integration being instrumented?" It's intended to aid compatibility checks, or alternative implementations of integrations, if they are version sensitive.

In most cases, this is a gem, so the Gem version suffices. Here, since the GC is a function of the platform and Ruby version, you could probably just put in `Gem::Version.new(RUBY_VERSION)` or something like that. Then if this extension is only compatible with particular versions of Ruby, you might want to express that in the `compatible?` function with something like `version >= Gem::Version.new('2.0')`.